test/log/log.go

```diff
@@ -0,0 +1,27 @@
+package log
```

Comment:
All this is already implemente by think go 

For example Step would be By("openning connection...  blahblah")